nomad/state/state_store_test.go

```diff
@@ -3522,8 +3522,8 @@ func TestStateStore_CSIPluginMultiNodeUpdates(t *testing.T) {
 
 func TestStateStore_CSIPluginJobs(t *testing.T) {
 	s := testStateStore(t)
-	deleteNodes := CreateTestCSIPlugin(s, "foo")
-	defer deleteNodes()
+	// deleteNodes := CreateTestCSIPlugin(s, "foo")
+	// defer deleteNodes()
```

Comment:
Was this just left behind from testing?

Reply:
yes, that was in place before jobs created plugins successfully